### PR TITLE
Update gcloud pubsub to v0.95

### PR DESCRIPTION
**Pull request by geyer-za**
Update google/cloud library from v0.35 to v0.95

### CODE REVIEW

**Comment by geyer-za** [comment]
Unit tests are passing

```# ./vendor/bin/phpunit ./tests/GoogleCloudPubSubAdapterTest.php
PHPUnit 5.7.27 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.0.33
Configuration: /opt/php-pubsub/phpunit.xml
Error:         No code coverage driver is available

................                                                  16 / 16 (100%)

Time: 65 ms, Memory: 6.00MB

OK (16 tests, 10 assertions)```

**Comment by nicja** [comment]
You should plug these tests into Travis as per our other OSS repos :+1:

**Review by nicja — CHANGES_REQUESTED** [review]
Not sure why tests are not running against TravisCI anymore? Also please update changelog for release, this should be a minor release as it's a minor release upgrade to the upstream package

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 5.0.2 - 2019-02-28
```
Comment (nicja) [inline changelog.md]: Should be a minor version increase, not patch

**Review by nicja — CHANGES_REQUESTED** [review]

**Review by nicja — APPROVED** [review]

**Pull request merged**